pkg/controller/nodenetworkconfigurationpolicy/enactmentconditions/conditions.go

```diff
@@ -1,24 +1,18 @@
 package enactmentconditions
```

Comment:
Conditions are a part of status. Could we just move this module under `nodenetworkconfigurationpolicy/enactmentstatus/conditions.sh`?

Reply:
Also rename "New" to "NewConditions"

Reply:
Since we have  conditions.go and counter.go what about moving both to enactmentstatus/conditions/ ?

Reply:
I guess it makes sense, but I can't say i have the current project structure in my head. If it feels more natural to you, then go for it.